src/test/java/jimmy/mcgymmy/model/person/UniquePersonListTest.java

```diff
@@ -1,148 +1,151 @@
-package seedu.address.model.person;
+package jimmy.mcgymmy.model.person;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
-import static seedu.address.testutil.Assert.assertThrows;
-import static seedu.address.testutil.TypicalPersons.ALICE;
-import static seedu.address.testutil.TypicalPersons.BOB;
 
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 
 import org.junit.jupiter.api.Test;
 
-import seedu.address.model.person.exceptions.DuplicatePersonException;
-import seedu.address.model.person.exceptions.PersonNotFoundException;
+import jimmy.mcgymmy.model.person.exceptions.DuplicatePersonException;
+import jimmy.mcgymmy.model.person.exceptions.PersonNotFoundException;
+import jimmy.mcgymmy.testutil.Assert;
+import jimmy.mcgymmy.testutil.TypicalPersons;
 
 public class UniquePersonListTest {
 
     private final UniquePersonList uniquePersonList = new UniquePersonList();
 
     @Test
     public void contains_nullPerson_throwsNullPointerException() {
-        assertThrows(NullPointerException.class, () -> uniquePersonList.contains(null));
+        Assert.assertThrows(NullPointerException.class, () -> uniquePersonList.contains(null));
```

Comment:
I think we should keep this as just `assertThrows` instead. We should stick to the static asserts for tests

Reply:
same for all the other test files that were changed

Reply:
> I think we should keep this as just `assertThrows` instead. We should stick to the static asserts for tests

Our `Assert` class only has the `assertThrows` method. I think it is fine for this case since we are not really unnecessarily importing. Also this was automatically optimised by intelliJ.